public/app/features/explore/LogsVolumePanelList.tsx

```diff
@@ -0,0 +1,127 @@
+
+import { LogsVolumePanel } from './LogsVolumePanel';
+import { SupplementaryResultError } from './SupplementaryResultError';
+
+type Props = {
+  logsVolumeData: DataQueryResponse | undefined;
+  absoluteRange: AbsoluteTimeRange;
+  timeZone: TimeZone;
+  splitOpen: SplitOpen;
+  width: number;
+  onUpdateTimeRange: (timeRange: AbsoluteTimeRange) => void;
+  onLoadLogsVolume: () => void;
+  onHiddenSeriesChanged: (hiddenSeries: string[]) => void;
+  eventBus: EventBus;
+};
+
+export const LogsVolumePanelList = ({
+  logsVolumeData,
+  absoluteRange,
+  onUpdateTimeRange,
+  width,
+  onLoadLogsVolume,
+  onHiddenSeriesChanged,
+  eventBus,
+  splitOpen,
+  timeZone,
+}: Props) => {
+  const logVolumes = useMemo(
+    () => groupBy(logsVolumeData?.data || [], 'meta.custom.sourceQuery.refId'),
+    [logsVolumeData]
+  );
+
+  const styles = useStyles2(getStyles);
+
+  const numberOfLogVolumes = Object.keys(logVolumes).length;
+
+  const containsZoomed = Object.values(logVolumes).some((data: DataFrame[]) => {
+    const zoomRatio = logsLevelZoomRatio(data, absoluteRange);
+    return !isLogsVolumeLimited(data) && zoomRatio && zoomRatio < 1;
+  });
+
+  let zoomedInfo;
+  if (containsZoomed) {
+    zoomedInfo = (
+      <InlineField label="Reload log volume" transparent>
+        <Button size="xs" icon="sync" variant="secondary" onClick={onLoadLogsVolume} id="reload-volume" />
+      </InlineField>
+    );
+  }
+
+  let extraInfo = <>{zoomedInfo}</>;
+
+  if (logsVolumeData?.state === LoadingState.Loading) {
+    return <span>Loading...</span>;
+  }
+
+  if (logsVolumeData?.error !== undefined) {
+    return <SupplementaryResultError error={logsVolumeData.error} title="Failed to load log volume for this query" />;
+  }
+
+  return (
+    <div className={styles.listContainer}>
+      {Object.keys(logVolumes).map((name, index) => {
+        const logsVolumeData = { data: logVolumes[name] };
+        return (
+          <LogsVolumePanel
+            key={index}
+            absoluteRange={absoluteRange}
+            width={width}
+            logsVolumeData={logsVolumeData}
+            onUpdateTimeRange={onUpdateTimeRange}
+            timeZone={timeZone}
+            splitOpen={splitOpen}
+            onLoadLogsVolume={onLoadLogsVolume}
+            // TODO: Support filtering level from multiple log levels
+            onHiddenSeriesChanged={numberOfLogVolumes > 1 ? () => {} : onHiddenSeriesChanged}
+            eventBus={eventBus}
+          />
+        );
+      })}
+      <div className={styles.extraInfoContainer}>{extraInfo}</div>
+    </div>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    listContainer: css`
+      padding-top: 10px;
+    `,
+    extraInfoContainer: css`
+      display: flex;
+      justify-content: end;
+      position: absolute;
+      right: 5px;
+      top: 5px;
+    `,
+    oldInfoText: css`
+      font-size: ${theme.typography.size.sm};
```

Comment:
`theme.typography.size` is deprecated:
```suggestion
      font-size: ${theme.typography.bodySmall.fontSize};
```

Reply:
Updated in 559805ce52856e2eab1f823f67f5226781517b68